alembic/env.py

```diff
@@ -0,0 +1,76 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config, pool
+from sqlmodel import SQLModel
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
```

Comment:
Here you want to configure the db:

```
config.set_main_option("sqlalchemy.url", DATABASE_URL)
```

see here: https://alembic.sqlalchemy.org/en/latest/api/config.html#alembic.config.Config